src/optimization/hooke_jeeves_method.cpp

```diff
@@ -0,0 +1,149 @@
+#include <k52/optimization/hooke_jeeves_method.h>
+
+#ifdef BUILD_WITH_MPI
+
+#include <boost/mpi.hpp>
+#include <k52/parallel/mpi/constants.h>
+
+#endif
+
+#include <iostream>
+#include <cmath>
+#include <algorithm>
+#include <stdexcept>
+
+#include <k52/optimization/params/i_continuous_parameters.h>
+
+using ::std::vector;
+
+namespace
+{
+	double kAcceleration = 3.5;
```

Comment:
Well, I suppose to use exactly this value every time but I can implement this value as class field and initialize it in constructor, for example. Should I realize it in this way?